### PR TITLE
Upgrade to Laravel 7

### PR DESCRIPTION
**Pull request by szainmehdi**
Lots of dependencies upgraded to support Laravel 7.

### CODE REVIEW

File: api/config/cors.php [hunk 1]
```diff
@@ -3,66 +3,39 @@
...
+    | To learn more: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
+    |
+    */
+
+    'paths' => ['*'],
+
+    'allowed_methods' => ['*'],
+
+    'allowed_origins' => [
+        'https://localhost:8080',
```
Comment (shea786) [inline api/config/cors.php:23]: @szainmehdi do we need localhost:8080 listed here as our domain structure for frontend requires the site to run off of nawhas.test:8080, maybe localhost:8080 should be replaced with nawhas.test:8080 ?

File: api/config/sanctum.php [hunk 2]
```diff
@@ -1,5 +1,13 @@
 <?php
 
+$local = env('APP_ENV') === 'local' ? [
+    env('APP_DOMAIN') . ':8080',
+    'localhost',
```
Comment (shea786) [inline api/config/sanctum.php:5]: I guess this should also be nawhas.test ?

File: api/routes/web.php [hunk 3]
```diff
@@ -15,3 +16,6 @@
     ->middleware(EnforceFeatureFlags::in([
         Features\SocialAuthentication::NAME,
     ]));
+
+// Backwards compatibility with Airlock
+Route::get('/airlock/csrf-cookie', [CsrfCookieController::class, 'show']);
```
Comment (shea786) [inline api/routes/web.php:21]: Should `airlock/csrf-cookie` not be `/sanctum/csrf-cookie` ?

**Comment by shea786** [comment]
Also i think we need to add the relevant middleware changes to the audit related routes too which is not merged into develop yet

File: api/config/cors.php (hunk 1 above)
Comment (szainmehdi) [reply api/config/cors.php:23]: I left these there for the sake of preserving defaults set by Laravel.

File: api/config/sanctum.php (hunk 2 above)
Comment (szainmehdi) [reply api/config/sanctum.php:5]: The `env('APP_DOMAIN')` bit takes care of the `nawhas.test` scenario

File: api/routes/web.php (hunk 3 above)
Comment (szainmehdi) [reply api/routes/web.php:21]: This is to preserve backwards compatibility with old versions of the frontend that will still call `/airlock/csrf-cookie`.

The `sanctum` route is automatically registered by Sanctum

**Pull request merged**